modules/system/lang/fr/lang.php

```diff
@@ -12,11 +12,6 @@
     'file' => [
         'create_fail' => "Impossible de créer le fichier : :name",
     ],
-    'page' => [
-        'invalid_token' => [
-            'label' => 'La clé de sécurité est invalide',
-        ],
-    ],
```

Comment:
Why was this removed?

Reply:
The `page` exist twice in the file.
https://github.com/wintercms/winter/blob/b1d28d9d3ef3e5889bf93a38dfbcfbabb723f256/modules/system/lang/fr/lang.php#L15-L19
And 
https://github.com/wintercms/winter/blob/b1d28d9d3ef3e5889bf93a38dfbcfbabb723f256/modules/system/lang/fr/lang.php#L410-L423